starknet_py/tests/e2e/deploy/deploy_test.py

```diff
@@ -1,96 +0,0 @@
-import pytest
-
-from starknet_py.contract import Contract, ContractFunction
-
-
-@pytest.mark.asyncio
-async def test_deploy_tx(gateway_account_client, map_compiled_contract):
-    result = await Contract.deploy(
-        client=gateway_account_client, compiled_contract=map_compiled_contract
-    )
-    result = await result.wait_for_acceptance()
-    result = result.deployed_contract
-
-    assert isinstance(result.functions["get"], ContractFunction)
-    assert isinstance(result.functions["put"], ContractFunction)
-
-
-@pytest.mark.asyncio
-async def test_deploy_with_search_path(gateway_account_client, base_compiled_contract):
-    result = await Contract.deploy(
-        client=gateway_account_client,
-        compiled_contract=base_compiled_contract,
-    )
-    await result.wait_for_acceptance()
-
-    result = await Contract.deploy(
-        client=gateway_account_client,
-        compiled_contract=base_compiled_contract,
-    )
-    result = await result.wait_for_acceptance()
-    result = result.deployed_contract
-    assert isinstance(result.functions["put"], ContractFunction)
-
-
-@pytest.mark.asyncio
-async def test_constructor_arguments(
-    gateway_account_client, constructor_with_arguments_compiled_contract
-):
-    value = 10
-    tuple_value = (1, (2, 3))
-    arr = [1, 2, 3]
-    struct = {"value": 12, "nested_struct": {"value": 99}}
-
```

Comment:
I wonder if we should have test verifying data transformer like this but with deploy_account or something else? We do have unit tests for data transformer so perhaps it's unnecessary though.

Reply:
I don't think such tests are necessary since `DataTransformer` tests are pretty extensive